student.library/src/main/java/tds/student/sql/data/OpportunityStatusExtensions.java

```diff
@@ -13,12 +13,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import tds.exam.ExamStatusCode;
-
 public class OpportunityStatusExtensions
 {
   private static final Map<String, OpportunityStatusType> _statuses = new HashMap<> ();
-  private static final Map<String, OpportunityStatusType> examStatusToOpportunityStatus = new HashMap<>();
```

Comment:
Yes.  These are just lookup tables that convert exam status code Strings to the existing legacy codes object types.